branded_ios_app/publishing_ios_app_8.rst

```diff
@@ -1,37 +1,45 @@
-================================
-Testing Your New Branded iOS App
-================================
-
-You'll distribute the file with the :file:`.ipa` extension, like our example 
-:file:`MyBiz iOS App-3.4.211.ipa`, from your 
-`<https://customer.owncloud.com/owncloud>`_ 
-account to your beta testers. To do this you'll need a Mac computer, an iPhone 
-or iPad registered in your Apple developer account, and the iTunes account 
-associated with your Apple developer account.
-
-1. Connect your registered iPhone or iPad to a Mac running iTunes.
-2. Double-click your iOS :file:`.ipa` file.
-3. You should see your device in the upper left corner of your iTunes windows. 
-   Click on it.
-4. Click the Apps button. Now you should see your app in the iTune apps list, 
-   with an Install button. Click it.
-5. The Install button changes to Will Install.
-6. Click the sync button in the lower-right corner to sync your device. This 
-   installs your app on your device.
-
-Your other testers can now install and test your app on their registered iPhones 
-and iPads just like any other app from iTunes. 
-
-.. Note:: If you have the Enterprise Apple developer account, there is no limit 
-   on the number of testing devices, and they do not have to be registered.
-
-Getting Crash Reports From Testers
+===============================
+Additional Server Configuration
+===============================
+
+Support for Apple Universal Links:
 ----------------------------------
 
-iOS automatically records crash logs when apps crash. Your testers can retrieve 
-and send these logs to you. They must follow these steps:
+There's some special changes that need to be made. Quoting from `Apple's official documentation on Universal Link Support
+<https://developer.apple.com/library/content/documentation/General/Conceptual/AppSearch/UniversalLinks.html>`_:
+
+ Adding support for universal links is easy. There are three steps you need to take:
+
+    - Create an apple-app-site-association file that contains JSON data about the URLs that your app can handle.
+    - Upload the apple-app-site-association file to your HTTPS web server. You can place the file at the root of your server or in the .well-known subdirectory.
+    - Prepare your app to handle universal links. 
+
+The `apple-app-site-association` will be generated by ownbrander and needs to be served statically on your site through HTTPS. You can safely place it on the root folder of your ownCloud installation (e.g. `/var/www/owncloud`). To achieve the second requirement, some changes will also need to be done on your Apache configuration.
+
+If you configured your installation with the official `Admin Manual
+<https://doc.owncloud.org/server/10.0/admin_manual/installation/source_installation.html#apache-configuration-label>`_, your `owncloud.conf` file should include the following:
```

Comment:
Must be a xref not a web link.
For details see best practices in readme.md

Reply:
This needs to be converted to AsciiDoc and moved to the `master-antora` branch. I'll get it sorted.